experiments/standalone/Bucket/global_bucket_temporalmap.jl

```diff
@@ -38,6 +38,14 @@ using ClimaLand:
     PrescribedRadiativeFluxes
 using ClimaLand.TimeVaryingInputs
 
+PROFILING = false
+try
```

Comment:
why is this in a try catch? so we only profile on buildkite when we've added ProfileCanvas to the environment?

Reply:
I don't want to add ProfileCanvas to experiments environment, and users should be able to run the code outside of buildkite